Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs

```diff
@@ -0,0 +1,31 @@
+using Doppler.BigQueryMicroservice.Repository;
+using Doppler.BigQueryMicroservice.Serialization;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace Doppler.BigQueryMicroservice.Controllers
+{
+    /// <summary>
+    /// Big query Api.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    public class BigQueryController : ControllerBase
+    {
+        private readonly IUnitOfWork unitOfWork;
+        public BigQueryController(IUnitOfWork unitOfWork)
+        {
+            this.unitOfWork = unitOfWork;
+        }
+
+        [HttpGet("/big-query/{email}/allow-emails")]
```

Comment:
We usually use `accountName` instead of `email`, but I'm not sure if that apply here.

Reply:
suggest accepted and applied